docs/source/chapters/about.rst

```diff
@@ -14,9 +14,8 @@ About The Workshop
     :align: center
     :target: http://www.osgeo.org/
 
-This workshop uses several FOSS4G tools. Most of FOSS4G software is
-related to other open source projects and it would go too far to list them all
-here.
+This workshop uses several FOSS4G (Free and Open Source Software for Geospatial) tools. Most of the FOSS4G tools are
```

Comment:
This workshop uses several ....
or
This workshop use several

Reply:
I think, it should be "This workshop use several...". Will be updated.